### PR TITLE
Moved Icarus Python dependencies to Icarus GitHub repo README file.

### PR DESCRIPTION
**Pull request by uhofemei**

### CODE REVIEW

File: dollhouse/README.md [hunk 1]
```diff
@@ -17,18 +17,10 @@ Node.js dependencies can be installed using `npm install <node_module>` (you nee
...
 * [Icarus](https://github.com/tripzero/icarus)
-* [Pysolar](http://pysolar.org/): version 0.6
-* [Pytz](http://pytz.sourceforge.net/): version 2015.7
-* [setuptools](https://pypi.python.org/pypi/setuptools): version 18.2
-* [Six](https://pypi.python.org/pypi/six): version 1.10.0 
-* [Twisted](https://twistedmatrix.com/): version 15.4.0
-* [txaio](https://pypi.python.org/pypi/txaio): version 2.0.4
-* [zope.interface](https://pypi.python.org/pypi/zope.interface): version 4.1.3
-
-## How to start the Home GW SW
```
Comment (gvancuts) [inline dollhouse/README.md]: Did you mean to remove this title line?

**Comment by gvancuts** [comment]
This is a good move, thanks!

Can you also add a small paragraph to indicate that if Ostro is used as a baseline (which we'd recommend) then some of the dependencies are already met (thinking about iotivity-node and mraa specifically, there may be others).

File: dollhouse/README.md (hunk 1 above)
Comment (uhofemei) [inline dollhouse/README.md]: I submitted a PR to the Icarus repo to add its dependencies there.

Comment (gvancuts) [inline dollhouse/README.md]: I saw that, but did you also mean to remove the title of the next section as part of that? (i.e.: ## How to start the Home GW SW)

**Comment by gvancuts** [comment]
Let's add the paragraph about Ostro as part of a later PR. Can you address the other pending question and let's finalise this.

**Comment by gvancuts** [comment]
Can you rebase your changes and address the open question?

**Comment by gvancuts** [comment]
@uhofemei ping?

**Comment by uhofemei** [comment]
I will get to it today.

**Pull request closed without merging**